samples/central-login/src/main.js

```diff
@@ -0,0 +1,99 @@
+import * as forgerock from '@forgerock/javascript-sdk';
```

Comment:
I like the changes here, but we have to make sure we update the SDK docs. So, we need to make sure both PRs (this and docs) get merged together.